crates/cxx-qt-gen/test_outputs/passthrough_and_naming.rs

```diff
@@ -208,6 +208,59 @@ pub mod ffi {
         #[doc(hidden)]
         fn cxx_qt_ffi_rust_mut(self: Pin<&mut SecondObject>) -> Pin<&mut SecondObjectRust>;
     }
+    #[namespace = ""]
+    unsafe extern "C++" {
+        type QPushButton;
+        #[namespace = "mynamespace"]
+        #[cxx_name = "ExternObjectCpp"]
+        type ExternObject;
+    }
+    unsafe extern "C++" {
+        #[rust_name = "clicked"]
+        fn clicked(self: Pin<&mut QPushButton>, checked: bool);
+    }
+    unsafe extern "C++" {
+        #[doc(hidden)]
+        #[namespace = "rust::cxxqtgen1::externcxxqt"]
+        #[must_use]
+        # [rust_name = QPushButton_connect_clicked]
+        fn QPushButton_clickedConnect(
+            self_value: Pin<&mut QPushButton>,
+            func: fn(Pin<&mut QPushButton>, checked: bool),
+            conn_type: CxxQtConnectionType,
+        ) -> CxxQtQMetaObjectConnection;
+    }
+    unsafe extern "C++" {
+        #[rust_name = "data_ready"]
+        fn dataReady(self: Pin<&mut ExternObject>);
+    }
+    unsafe extern "C++" {
+        #[doc(hidden)]
+        #[namespace = "rust::cxxqtgen1::externcxxqt::mynamespace"]
+        #[must_use]
+        # [rust_name = ExternObject_connect_data_ready]
+        fn ExternObject_dataReadyConnect(
+            self_value: Pin<&mut ExternObject>,
+            func: fn(Pin<&mut ExternObject>),
+            conn_type: CxxQtConnectionType,
+        ) -> CxxQtQMetaObjectConnection;
+    }
+    unsafe extern "C++" {
+        #[rust_name = "error_occurred"]
+        #[rust_name = "errorOccurred"]
+        fn errorOccurred(self: Pin<&mut ExternObject>);
+    }
+    unsafe extern "C++" {
+        #[doc(hidden)]
+        #[namespace = "rust::cxxqtgen1::externcxxqt::mynamespace"]
+        #[must_use]
+        # [rust_name = ExternObject_connect_error_occurred]
```

Comment:
This will probably cause a clippy lint or even compiler warning about "function name should be snake_case"

Reply:
Even using #670 and running clippy in the example doesn't complain :thinking:

Reply:
Alright, then it's fine.
We can always fix if we do find anything.